qemu/tests/secure_execution.py

```diff
@@ -0,0 +1,44 @@
+from virttest import error_context
+from virttest import virt_vm
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Secured guest boot test:
+    1) Log into a secure guest
+    2) Check if there's error messagein
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+
+    timeout = float(params.get("login_timeout", 240))
```

Comment:
```suggestion
```

Reply:
not get the point, why do I need to delete the timeout here, is it find if I change it to timeout = params.get_numeric("login_timeout", 360)?

Reply:
removed